### PR TITLE
Proxy artwork and image sitemaps to S3 artsy-sitemaps.

### PR DESCRIPTION
**Pull request by dblock**
This gets rid of Fusion, see https://github.com/artsy/cinder/issues/86 for details. This also gets rid of bing sitemaps.

Remaining work to merge:

- [x] Check with #seo whether it's ok to remove bing sitemaps.
- [x] Merge, deploy and schedule sitemap generation job in spark, https://github.com/artsy/cinder/pull/90. 

Robots.txt now contains `sitemap-arworks.xml` and `sitemap-images.xml`, the artworks are removed from `sitemaps.xml`. Note that it's not possible to have more than 1 level of nesting of sitemap indexes, so we cannot add indexes to the sitemaps.xml index.

We proxy sitemaps like `sitemap-artworks.xml`,  `sitemap-artworks-2017-01-02.xml` and `sitemap-artworks-2017-01-02-1.xml` and similar for `sitemap-images` to s3://artsy-sitemaps instead of querying buckets of artworks from fusion.

### CODE REVIEW

File: desktop/config.coffee [hunk 1]
```diff
@@ -94,7 +94,6 @@ module.exports =
   TEAM_BLOGS: '^\/life-at-artsy$|^\/artsy-education$|^\/gallery-insights$'
   MARKETING_SIGNUP_MODALS: '[{"slug":"ca1","copy":"An art collection for every budget","image":"http://files.artsy.net/images/modal-interior-a-credited.jpg","photoCredit":"Photo by Emily Johnson"},{"slug":"ca2","copy":"Buy art from the best galleries and auction houses","image":"http://files.artsy.net/images/modal-interior-a-credited.jpg","photoCredit":"Photo by Emily Johnson"},{"slug":"ca3","copy":"An art collection for every budget","image":"http://files.artsy.net/images/modal-interior-b-credited.jpg","photoCredit":"Photo by Emily Johnson"},{"slug":"ca4","copy":"Buy art from the best galleries and auction houses","image":"http://files.artsy.net/images/modal-interior-b-credited.jpg","photoCredit":"Photo by Emily Johnson"}]'
   EOY_2016_ARTICLE: null
-  EOY_2016_TEASER: 'https://artsy-vanity-files-production.s3.amazonaws.com/documents/year-in-art-teaser.html'
```
Comment (dblock) [inline desktop/config.coffee]: Couldn't find a reference to it so deleted. Maybe there're others ...

Comment (kanaabe) [inline desktop/config.coffee]: 👍🏼👍🏼good to remove, thanks!

File: desktop/apps/sitemaps/index.coffee [hunk 2]
```diff
@@ -7,14 +7,11 @@ app.set 'view engine', 'jade'
...
-app.get '/sitemap-artworks-:page-:date.xml', routes.artworksPage('artworks')
-app.get '/sitemap-images-:page-:date.xml', routes.artworksPage('images')
+app.get ['/sitemap-artworks.xml', '/sitemap-artworks-:timestamp.xml'], routes.sitemaps
+app.get ['/sitemap-images.xml', '/sitemap-images-:timestamp.xml'], routes.sitemaps
+app.get '/sitemap-articles-:page.xml', routes.articlesPage # archive of all articles (for main sitemap, not news-specific sitemap)
 app.get '/sitemap-:resource-:page.xml', routes.resourcePage
-app.get '/sitemap-cities.xml', routes.cities
-app.get '/images_sitemap.xml', routes.imagesIndex
-app.get '/sitemap-bing-images.json', routes.bingjson
-app.get '/sitemap-bing-images-new.json', routes.bingNew
```
Comment (kanaabe) [inline desktop/apps/sitemaps/index.coffee]: 👏

File: desktop/apps/sitemaps/routes.coffee [hunk 3]
```diff
@@ -1,21 +1,18 @@
...
 PAGE_SIZE = 100
-FUSION_PAGE_SIZE = 5000
 
-epoch = -> moment('2010 9 1', 'YYYY MM DD')
-buckets = _.times moment().diff(epoch(), 'months'), (i) ->
-  {
-    start: epoch().add(i, 'months').format('YYYY-MM-DD')
-    end: epoch().add(i + 1, 'months').format('YYYY-MM-DD')
-  }
+{ SITEMAP_BASE_URL } = require('../../config')
```
Comment (kanaabe) [inline desktop/apps/sitemaps/routes.coffee]: Assuming this is sensitive -- if not we can add to [sharify](https://github.com/artsy/force/blob/master/desktop/lib/setup_sharify.coffee#L77)

Comment (dblock) [inline desktop/apps/sitemaps/routes.coffee]: It's not sensitive. I tried to add it to setup_sharify and I don't see it here still

Comment (dblock) [inline desktop/apps/sitemaps/routes.coffee]: Nope, not sensitive, fixed.

File: desktop/apps/sitemaps/routes.coffee [hunk 4]
```diff
@@ -147,87 +92,11 @@ getArtworkBuckets = (callback) ->
...
-    json = _.extend json, {
-      "height": dimensions.height
-      "width": dimensions.width
-    }
-  json
+# sitemaps for artworks and images are generated in Spark by Cinder and written to the artsy-sitemaps s3 bucket
+# see https://github.com/artsy/cinder/blob/master/doc/sitemaps.md for more information
+@sitemaps = (req, res, next) ->
+  req.headers['host'] = sitemap_host
+  sitemapProxy.web req, res
```
Comment (kanaabe) [inline desktop/apps/sitemaps/routes.coffee]: 👍

**Pull request merged**